packages/mds-policy-service/repository/entities/policy-entity.ts

```diff
@@ -30,4 +32,7 @@ export class PolicyEntity extends IdentityColumn(class {}) implements PolicyEnti
 
   @Column('json')
   policy_json: PolicyEntityModel['policy_json']
+
+  @Column('uuid', { nullable: true, array: true })
+  superseded_by: PolicyEntityModel['superseded_by']
```

Comment:
Oo intreesting, maybe in the future I also get PolicyDomainModel superseded_by, so that I may resolve policy->supercededBy: PolicyDomainModel[] from a uuid[]

Reply:
Yup, in the future this could totally be something we expose optionally from the domain model :)